canonical-web-and-design/vanilla-framework
# Chips

## Done

- Add maps with semantic colour tinted backgrounds and borders 
- Use those and some color adjustments to derive chip states

(This is meant to replace Scott's pr).

Fixes #4013 

## QA

- Open https://vanilla-framework-4124.demos.haus/docs/examples/patterns/chip/variants
- Verify it looks like the screenshot below. Test states. 
![image](https://user-images.githubusercontent.com/2741678/142027109-18bb63fd-38df-49fb-9b22-a6cb24ecfcab.png)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
